src/assets/sass/demos/article.scss

```diff
@@ -0,0 +1,567 @@
+// Define paths
+$font-path:  '/assets/protocol/protocol/fonts';
+$image-path: '/assets/protocol/protocol/img';
+
+// Protocol core
+@import '../protocol/includes/lib',
+        '../protocol/protocol';
+
+// Protocol components
+@import
+    '../protocol/templates/main-with-sidebar',
+    '../protocol/components/article',
+    '../protocol/components/sidebar-menu';
+
+
+// Article links... is this a component?
```

Comment:
This was a holdover from the security page this was originally based on, and on that page at least this set of links is a one-off thing. We'll have to figure out what to do about that when we're actually reskinning the security pages in bedrock, but I think for now I can just delete this bit here since we're not using it in the demo.